test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesTestHelpers.cs

```diff
@@ -50,7 +51,7 @@ public static async Task<HttpResponseMessage> InvokeDrain(EndToEndTestFixture fi
         {
             string masterKey = await fixture.Host.GetMasterKeyAsync();
             string uri = $"admin/host/drain?code={masterKey}";
-            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, uri);
```

Comment:
Strange that Drain supports both GET/POST, but Resume only supports POST. Guess this was an unnecessary addition when Drain was added? If there was no reason why GET is supported (looks like infra only uses POST), I guess it's fine that Resume does the right thing and only supports POST, though it is a discrepancy

Reply:
Yeah ideally we want to remove GET support from drain, I spoke to the Atlas and Antares teams and they're both using POST so I think it is safe to get rid of it but first we just want to run a query to see if any GET requests are coming in at all from anywhere. Will address in a new PR.

Reply:
New PR to deal with some drain mode cleanup, resolving this comment and we can discuss here if there are issues!

https://github.com/Azure/azure-functions-host/pull/7876/files